### ISSUE TITLE
Code for evaluation on MEGC2021 benchmark?

### ISSUE THREAD

**Issue by xjtupanda** [genbing99/MTSN-Spot-ME-MaE#3]
Thanks for the great work. I noticed that there's only evaluation code for SAMM-Challenge and CAS(ME)^3, would you please also provide the counterpark for SAMM-LV and CAS(ME)^2 for reproduction on the F1-score results for those datasets?

**Comment by genbing99** [comment]
Hi, I have lost the network weights to reproduce the results that are shared in the paper. 
Due to time constraints, I tried to rerun the codes again with other configurations, but the results obtained are lower than the reported ones. Hope you can understand.
The jupyter notebook file which consists of all steps is in the link:
https://drive.google.com/file/d/16sUEb3RmIaDh-IvrcT6L9sRHGuG-t6mJ/view?usp=sharing
Note that I used a high learning rate in this notebook, so the val_loss values in the graph fluctuated. A slightly better result and a nicer loss graph can be obtained if a lower learning rate is used.

**Comment by xjtupanda** [comment]
> Hi, I have lost the network weights to reproduce the results that are shared in the paper. Due to time constraints, I tried to rerun the codes again with other configurations, but the results obtained are lower than the reported ones. Hope you can understand. The jupyter notebook file which consists of all steps is in the link: https://drive.google.com/file/d/16sUEb3RmIaDh-IvrcT6L9sRHGuG-t6mJ/view?usp=sharing

Thank you so much for your patience and kindness!
